main/movies.js

```diff
@@ -0,0 +1,143 @@
+        country: "USA",
+        genre: "thriller"
+        
+    },
+    {
+        id:4,
+        name: "SuiriyasKanguva",
+        img: "top5/img 4.jpg",
+        description: "The film was reannounced in August that year with the tentative title Suriya 42 as it became Suriya's 42nd film in a leading role. Principal photography commenced that month in Chennai and ended in January 2024. It was filmed in various locations including Goa, Kerala, Kodaikanal and Rajahmundry. The title Kanguva was announced in April 2023. The technical crew includes Devi Sri Prasad, Vetri Palanisamy and Nishadh Yusuf, as music composer, cinematographer and editor, respectively. Made on a budget of around ₹300–350 crore, Kanguva is one of the most expensive Indian films to date",
+        year: "2024",
+        director: " Siva",
+        producer: "	K. E. Gnanavel Raja",
+        writer: "Adi Narayana",
+        trailerURL: "https://www.youtube.com/embed/oBlxdr1KbEA?si=NRzx7IIjVg8W-T_m",
+        country: "India",
+        genre: "fantasy"
+        
+    },
+    {
+        id:5,
+        name: "CaptainMiller",
+        img: "top5/img 2.jpg",
+        description: "The film was reannounced in August that year with the tentative title Suriya 42 as it became Suriya's 42nd film in a leading role. Principal photography commenced that month in Chennai and ended in January 2024. It was filmed in various locations including Goa, Kerala, Kodaikanal and Rajahmundry. The title Kanguva was announced in April 2023. The technical crew includes Devi Sri Prasad, Vetri Palanisamy and Nishadh Yusuf, as music composer, cinematographer and editor, respectively. Made on a budget of around ₹300–350 crore, Kanguva is one of the most expensive Indian films to date",
+        year: "2024",
+        director: "Arun Matheswaran",
+        producer: "Sendhil Thyagarajan",
+        writer: "Arun Matheswaran",
+        trailerURL: "https://www.youtube.com/embed/ujhWbKP1rKA?si=SXDa0xkFnbCp5S",
+        country: "India",
+        genre: "action,adventure"
+        
+    },
+
+]
+    
+
+const currentFilm = moviesData.find(el => el.name === filmName)
+
+
+const {name, img, description, year, director, producer, writer, trailerURL, country, genre} = currentFilm
+
+document.body.style.backgroundImage = `url("${img}")`
+currCountry.innerHTML = country;
+currGenre.innerHTML = genre;
+currDirector.innerHTML = director;
+currProducer.innerHTML = producer;
+currWriter.innerHTML = writer;
+currYear.innerHTML = year;
+currPic.src = img;
+currDescription.innerHTML = description;
+currIframe.src = trailerURL;
+
+
+
+    if(logedUserData !== null) {
+
+            for(let i = 1;i <= 50;i++) {
+                const container = document.createElement("div");
+                container.classList.add("boxes")
+                container.innerHTML = i;
+                
+                if(logedUserData.tickets[filmName].includes(i)) {
```

Comment:
should be replaced tickets finding logic. 

Separate tickets const and iterate over it.